tests/snippets/delete.py

```diff
@@ -10,3 +10,6 @@ class MyObject: pass
 
 assert not hasattr(foo, 'bar')
 
+x = 1
+y = 2
+del (x, y)
```

Comment:
Could you add a test using `assert_raises` ensuring that the variables actually have been deleted?